pacman_mirrors/pacman_mirrors.py

```diff
@@ -128,7 +116,7 @@ def command_line_parse(self):
                             type=int,
                             metavar=txt.SECONDS,
                             help=txt.HLP_ARG_TIMEOUT)
-        parser.add_argument("--no-update",
+        parser.add_argument("--no_update",
```

Comment:
You're absolute right - had no intentions regarding that - thanks for spotting it :)
But I think I spotted it before you because my version is correct.